.github/workflows/lint.yml

```diff
@@ -0,0 +1,91 @@
+---
+name: VM lint workflow
+on:
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
Should the various release branches be included? `**a` might catch any valid branches we care about. Alternatively, we could just do all branches.

Reply:
I don't think so. I just added push-master for the README badge, so it wouldn't reflect the status of the last PR touched. I don't think our history branches are generally lint clean at this point, and may get worse when tested against a 20.04 base